req/exporter.dim

```diff
@@ -144,3 +145,21 @@ SRS_DRAGNET_0067:
   tags:            covered, tested
   test_setups:     off_target
   status:          valid
+
+SRS_DRAGNET_0079:
```

Comment:
General question that popped up in my mind, not strictly related to this PR: is there a point in having these .dim requirements open sourced?

Reply:
Yes, of course.

Remember that Dragnet is a qualified software, it follows ISO-26262's guidelines, hence it needs to have requirements and they need to be tracked down. Right now we have no traceability checks because the tools we need for that aren't open source (yet). But eventually we will be able to have it in the open.

We need to keep maintaining the requirements until then.